code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -24,27 +24,36 @@ Bonus
 	level = 5
 	severity = 0
 
+/datum/symptom/sensory_restoration/proc/check_and_add(reagent, check, add, mob/living/M)
+	if(M.reagents.get_reagent_amount(reagent) < check)
+		M.reagents.add_reagent(reagent, add)
+		return 1
+
```

Comment:
It doesn't return a value if it didn't add. Should still work but something to keep in mind for later projects.

Reply:
Completely on purpose, to limit the amount of Notices in chat.

Reply:
I mean that you can use return 0 instead. Or return FALSE (and 1 will be TRUE). This way you always get a value when you call the proc. Now it will return NULL which will work in the current situation due to how DM uses the if statements.

Reply:
Ah, i see. Makes sense, thank you.